### DISCUSSION TITLE
Not able to receive message (FrozenError: can't modify frozen String)

### DISCUSSION THREAD

**Discussion by og14112014** [postalserver/postal#2269] [Help with using Postal]
I am able to send email to external emails like Gmail but not able to send to same domain or able to receive it.
Below are logs

postal-worker-1    | [9] [2022-12-03T09:59:03.575] INFO -- : [OASOJ53J] Connected to ****:25 (mx.postal.**.email)
postal-worker-1    | [9] [2022-12-03T09:59:03.576] INFO -- : [OASOJ53J] Sending message 1::9 to admin@**.email
postal-worker-1    | [9] [2022-12-03T09:59:03.605] INFO -- : [OASOJ53J] ArgumentError: mail destination not given
postal-worker-1    | [9] [2022-12-03T09:59:03.611] INFO -- : [AVK4E9RVXX] [1::9 9] Internal error: FrozenError: can't modify frozen String
postal-worker-1    | [9] [2022-12-03T09:59:03.611] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/message_db/delivery.rb:53:in `force_encoding'
postal-worker-1    | [9] [2022-12-03T09:59:03.611] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/message_db/delivery.rb:53:in `webhook_hash'
postal-worker-1    | [9] [2022-12-03T09:59:03.611] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/message_db/delivery.rb:44:in `send_webhooks'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/message_db/delivery.rb:11:in `create'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/message_db/message.rb:128:in `create_delivery'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/app/jobs/unqueue_message_job.rb:419:in `block in perform'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/app/jobs/unqueue_message_job.rb:30:in `each'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/app/jobs/unqueue_message_job.rb:30:in `perform'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/worker.rb:55:in `receive_job'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /opt/postal/app/lib/postal/worker.rb:88:in `block in join_queue'
postal-worker-1    | [9] [2022-12-03T09:59:03.614] INFO -- : [AVK4E9RVXX] [1::9 9] /usr/local/bundle/gems/bunny-2.14.4/lib/bunny/consumer.rb:56:in `call'

**Comment by willpower232** [comment]
Have a look at #2153
